__tests__/app.test.js

```diff
@@ -111,3 +112,49 @@ describe("/api/articles", () => {
   })
   
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("get all comments for a specified article", () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({body}) => {
+      const {comments} = body;
+      expect(comments).toHaveLength(11);
+      comments.forEach(comment => {
+      expect(comment).toMatchObject({
+        comment_id: expect.any(Number),
+        body: expect.any(String),
+        article_id: expect.any(Number),
+        author: expect.any(String),
+        votes: expect.any(Number),
+        created_at: expect.any(String)
+      })  
+      })
+      });
+  });
+
+  test("GET:404 sends an appropriate status and error message when article id given is valid but non-existen id", () => {
+        return request(app)
+        .get("/api/articles/99/comments")
+        .expect(404)
+        .then((response) => {
+        expect(response.body.msg).toBe("this article doesn't have a comment");
```

Comment:
Dunno if this is a article that exists?  So maybe the error message should say that.